Essentials/src/main/java/com/earth2me/essentials/commands/Commandkitreset.java

```diff
@@ -0,0 +1,36 @@
+package com.earth2me.essentials.commands;
+
+import com.earth2me.essentials.User;
+import org.bukkit.Server;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Commandkitreset extends EssentialsCommand {
```

Comment:
This command should ideally support tab completion and running from the console.